modules/core/client/config/core.client.routes.js

```diff
@@ -13,10 +13,10 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
         url: '/',
         templateUrl: 'modules/users/views/authentication/signin.client.view.html'
       })
-      .state('myclasses', {
+      /**.state('myclasses', {
         url: '/myclasses',
         templateUrl: 'modules/core/views/home.client.view.html'
-      })
+      })*/
       .state('not-found', {
```

Comment:
you can delete this, instead of commenting

Reply:
@Bakushin10 This was left commented in case it needed to be referenced later.

Reply:
you created new module so I think you can delete it now. Also, did you check if home.client.view.html  used anywhere else?

Reply:
The file home.client.view.html still exists, but I think that it is the only place it is used. And I will go ahead and delete the commented out section.